### ISSUE TITLE
epresent should depend on org-8 in MELPA

### ISSUE THREAD

**Issue by pscollins** [eschulte/epresent#33]
I had problems trying to install epresent where I got the following error messages:

     Cannot open load file: ox

     Symbol's value as variable is void:	org-list-allow-alphabetical

Googling showed that it's because I was [missing](https://lists.gnu.org/archive/html/emacs-orgmode/2014-05/msg00703.html) the [latest version](http://stackoverflow.com/questions/16777426/emacs-org-mode-export-ox-html-ox-latex-cannot-open-load-file) of org-mode. It seemed to be fixed by `M-x package-install RET org RET` but this should be taken care of the package's dependencies --- I see that `org-present` includes a dependency on `org-7`, for example. 

If epresent does depend on `org-8` as it appears to, then `org-8` should be a dependency of the package.

**Comment by sellout** [comment]
I don’t think this is an issue any more – Org 8.2 is included with current Emacs versions.

**Comment by ipstone** [comment]
I have org 8.2, but package install epresent shows that org-8 is not present.

**Comment by adamjrat** [comment]
Apparently it _is_ still an issue! With a fresh install of emacs on updated centos7 and I hit this bug.  Thanks to pscollins for the workaround, and -1 to sellout for the copout on dependency management.
